### ISSUE TITLE
Feature: rename `config.yaml` to something else.

### ISSUE THREAD

**Issue by juzhiyuan** [api7/adc#59]
### Description

<img width="953" alt="image" src="https://github.com/api7/adc/assets/2106987/3e5af923-1246-4c6e-86c1-6713cf16b46e">

Hi, the `config.yaml` file is used to store non-API-related configurations, so it's wired to see it after conversion from OpenAPI.

**Comment by tao12345666333** [comment]
rename it to `api.yaml`, WDYT?

**Comment by tao12345666333** [comment]
Cc @navendu-pottekkat

**Comment by navendu-pottekkat** [comment]
A user could name the file anything. The only default naming we choose in ADC is the `.adc.yaml` configuration file for ADC. A user can have multiple configuration files with any names and pass them to ADC with the -f flag in any of the commands.

I think the question here is whether we should change the documentation to use a different name for the configuration files. If we are doing that, we should be consistent across the documentation of all subcommands.

**Comment by tao12345666333** [comment]
Make sense. 

I think it’s okay not to make any changes

**Comment by navendu-pottekkat** [comment]
@tao12345666333 Yes, me too. Because I don't know any other better names that could describe a configuration.

@juzhiyuan What do you think? Do you think we should still change it? Any suggestions?

**Comment by navendu-pottekkat** [comment]
How about `apisix.yaml`? Then in both scenarios, `apisix.yaml` is the route configuration for APISIX.
